src/asciidoc/docs/platform-features/richPreview.adoc

```diff
@@ -153,67 +71,18 @@ const SocialSharePreview = styled.div`
 `;
 ----
 
-.Vanilla JS
-[source,CSS]
-----
-#socialSharePreview {
-  background-color: #395b88;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  text-align: center;
-  color: #fcfcfc;
-}
-----
-
-.Angular
-[source,CSS]
-----
-#socialSharePreview {
-  background-color: #395b88;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  text-align: center;
-  color: #fcfcfc;
-}
-----
-
-.Svelte
-[source,CSS]
-----
-.social-share {
-  background-color: #395b88;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  text-align: center;
-  color: #fcfcfc;
-}
-----
-
---
-. To test the preview image, open your staging link and append `?koji-screenshot=1` to the URL.
+. You can preview this image by selecting menu:More[Thumbnail] in the preview pane.
```

Comment:
We should specify that we mean the preview pane of the Koji code editor (since they could be developing locally). Maybe also mention where to test it in the debugger.